corehq/apps/app_manager/tests/test_suite_remote_request.py

```diff
@@ -731,3 +731,19 @@ def test_allow_blank_value(self, *args):
         </partial>
         """
         self.assertXmlPartialEqual(expected, suite, "./remote-request[1]/session/query/prompt[@key='name']")
+
+    def test_exclude_from_search(self, *args):
+        self.module.search_config.properties[0].exclude = True
+        suite = self.app.create_suite()
+        expected = """
+        <partial>
+          <prompt key="name" exclude="true">
```

Comment:
I think this should be an xpath expression:
```suggestion
          <prompt key="name" exclude="true()">
```